AnkiDroid/src/main/java/com/ichi2/anki/reviewer/ActionButtonStatus.java

```diff
@@ -65,6 +66,13 @@ public void setup(SharedPreferences preferences) {
         setupButton(preferences, R.id.action_toggle_whiteboard, "customButtonEnableWhiteboard", SHOW_AS_ACTION_NEVER);
         setupButton(preferences, R.id.action_save_whiteboard, "customButtonSaveWhiteboard", SHOW_AS_ACTION_NEVER);
         setupButton(preferences, R.id.action_change_whiteboard_pen_color, "customButtonWhiteboardPenColor", SHOW_AS_ACTION_IF_ROOM);
+        if(Lookup.isAvailable()){  //checks if Lookup Dictionary preference is enabled
+            setupButton(preferences, R.id.action_search_dictionary, "customButtonSearchDictionary",SHOW_AS_ACTION_IF_ROOM);
+        }
+        else{
+            setupButton(preferences, R.id.action_search_dictionary, "customButtonSearchDictionary",MENU_DISABLED);
+        }
```

Comment:
Even with the correction, this isn't correct. We want to disable the button if lookup is disabled, but the final parameter is the default if the preference isn't set, so the button could still appear, even if lookup is disabled

```suggestion
         setupButton(preferences, R.id.action_search_dictionary, "customButtonSearchDictionary", Lookup.isAvailable() ? SHOW_AS_ACTION_IF_ROOM : MENU_DISABLED);
```

Reply:
can you please suggest a fix for this ?